src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,132 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_INITIAL_CAPACITY = 16;
+    private static final int RESIZE_FACTOR = 2;
+    private static final int NEXT_ENTRY_SIZE = 1;
+    private static final float LOAD_FACTOR = 0.75f;
+    private int size;
+    private int threshold;
+    private Node<K, V>[] table;
+
+    public MyHashMap() {
+        table = (Node<K, V>[]) new Node[DEFAULT_INITIAL_CAPACITY];
+        threshold = (int) (DEFAULT_INITIAL_CAPACITY * LOAD_FACTOR);
+    }
 
     @Override
     public void put(K key, V value) {
-
+        if (size + NEXT_ENTRY_SIZE > threshold) {
+            resize();
+        }
```

Comment:
remove this constant
```suggestion
        if (size >= table.length * LOAD_FACTOR) 
            resize();
       }
```